web-portal/backend/src/main.ts

```diff
@@ -5,15 +5,11 @@ import { AppModule } from './app.module';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   const { httpAdapter } = app.get(HttpAdapterHost);
-  app.useGlobalFilters(new PrismaClientExceptionFilter(httpAdapter));
+  // app.useGlobalFilters(new PrismaClientExceptionFilter(httpAdapter));
```

Comment:
Another instance of commenting out code, not a fan.

Reply:
understood ser, both are todo figure later!